docs/audit_report/src/01_generic_changes.rst

```diff
@@ -2,7 +2,54 @@ Changes Overview
 ================
 
 Since the previously audited version (|botan_git_base_ref|), Botan
-|botan_version| brings some extensions and fixes. The most relevant changes are outlined below.
+|botan_version| brings various extensions and fixes. The most relevant changes are outlined below.
 
-.. todo:: Outline the most notable changes that came with Botan 3.3.0
-          Presumably, that will be FrodoKEM and LMS or both.
+FrodoKEM
+--------
+
+FrodoKEM is a post-quantum key encapsulation algorithm based on unstructured
+lattices. In NIST's standardization process it is a "Round 3 alternate
+candidate", therefore it likely won't be standardized by NIST. However, it is
+recommended for usage by the BSI since 2020 [TR-02102-1]_.
```

Comment:
```suggestion
lattices. In NIST's standardization process, it is a "Round 3 alternate
candidate"; therefore, it likely won't be standardized by NIST. However, it has been
recommended for usage by the BSI since 2020 [TR-02102-1]_.
```